index.html

```diff
@@ -39,12 +39,405 @@ <h1 class="name">
+                     </div></a>
+                    </div>
+
+            </section>
+            <section class="page5">
+
+                <div class="page5-list" >
+                    <img class="laptop" src="./images/laptop.png" alt="laptop">
+                    <div class="rectangle">
+                    <p class="first-project"> Fifth Project </p>
+                   
+                    </div>
+                    <ul class="work-ul">
+                        <li >
+                            HTML/CSS
+                        </li>
+                                    <li>
+                                        Ruby on rails
+                                    </li>
+                               
+                                <li>
+                                    Java Script
+                                </li>
+                          
+                    </ul>
+                  <a href="#">  <div class="seeproject">
+                       <div class="see-project-icon">
+                        <img src="./images/ic_arrow_right.png" alt="Arrow" >
+                       </div>
+                       <div class="see-project-pargraph">
+                           <p>
+                               See project
+                           </p>
+                       </div>
+                     </div></a>
+                    </div>
+
+
+            </section>
+            <section class="page">
+
+                <div class="page-list" >
+                    <img class="laptop" src="./images/laptop2.png" alt="laptop">
+                    <div class="rectangle">
+                    <p class="first-project"> Sixth Project </p>
+                   
+                    </div>
+                    <ul class="work-ul">
+                        <li >
+                            HTML/CSS
+                        </li>
+                                    <li>
+                                        Ruby on rails
+                                    </li>
+                               
+                                <li>
+                                    Java Script
+                                </li>
+                          
+                    </ul>
+                  <a href="#">  <div class="seeproject">
+                       <div class="see-project-icon">
+                        <img src="./images/ic_arrow_right.png" alt="Arrow" >
+                       </div>
+                       <div class="see-project-pargraph">
+                           <p>
+                               See project
+                           </p>
+                       </div>
+                     </div></a>
+                    </div>
+
+
+            </section>
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+            <section class="page6">
+                <div class="about">
+                    <h1>
+                        About
+                    </h1>
```

Comment:
- [ ] Please, make sure The section `About` uses the font and colours (background and font) defined for the template.
**- Your design:**

![image](https://user-images.githubusercontent.com/80612925/177208900-3e3dd15f-d867-439d-bd03-059546bf810d.png)


**- Figma design:**

![image](https://user-images.githubusercontent.com/80612925/177208745-273fc03b-9b4d-4428-b65f-58f1f5d5f832.png)

Reply:
when I use diffrent font family it cause linter errors 

how do i use other than sans,sarif??